Libraries/Components/Button.js

```diff
@@ -25,107 +25,196 @@ import type {ColorValue} from '../StyleSheet/StyleSheet';
 
 type ButtonProps = $ReadOnly<{|
   /**
-   * Text to display inside the button
+    Text to display inside the button. On Android the given title will be converted to the uppercased form.
    */
   title: string,
 
   /**
-   * Handler to be called when the user taps the button
+    Handler to be called when the user taps the button. The first function argument is an event in form of [PressEvent](pressevent).
    */
   onPress: (event?: PressEvent) => mixed,
 
   /**
-   * If true, doesn't play system sound on touch (Android Only)
-   **/
+    If `true`, doesn't play system sound on touch.
+
+    @platform android
+
```

Comment:
Are these double line breaks intentional?

Reply:
Yeah it would help in parsing data easily. Following your suggestion to use markdown like format(double line for new line)